docs/rst/tutorials/cloud/wan_edge_cloud/wan_edge_cloud.rst

```diff
@@ -0,0 +1,280 @@
+Then, start by creating the workspace of this application and downloading the source code:
+
+.. code-block:: bash
+
+    mkdir -p turtlesim_move_ws/src && cd turtlesim_move_ws/src
+    git clone --branch humble https://github.com/eProsima/vulcanexus.git
+    mv vulcanexus/code/turtlesim .
+    rm -rf vulcanexus
+    cd ..
+
+Once created the workspace, source the Vulcanexus environment and build the ``turtlesim_square_move`` application.
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+    colcon build
+
+.. note::
+
+    Since the purpose of this tutorial is not to explain how to create a ROS 2 node, but rather, the communication of these in distributed environments, the code will not be discussed in detail.
+    Stay tuned for new tutorials in which we will discuss how to configure ROS 2 nodes, publishers/subscribers, services and actions.
+
+Now run the application:
+
+.. code-block:: bash
+
+    source turtlesim_move_ws/install/setup.bash
+    ros2 run turtlesim turtlesim_square_move
+
+The important points to note in this application are the following:
+
+*   The control application sends the movement commands to the ``turtlesim_node`` through a publisher in the `geometry_msgs/msg/Twist <https://docs.ros2.org/foxy/api/geometry_msgs/msg/Twist.html>`_ topic.
+    This topic expresses the velocity at which the turtle has to move divided into linear velocity (``linear``) and angular velocity (``angular``).
+
+*   The application knows the position of the turtle on the map at any moment and whether it is stopped or moving by subscribing to the `turtlesim/msg/pose <https://docs.ros2.org/foxy/api/turtlesim/msg/Pose.html>`_ topic.
+    This topic provides information about the turtle's coordinates (``x`` and ``y``) and the turtle's rotation (``theta``). We can also know its linear and angular velocity (``linear_velocity`` and ``angular_velocity``).
+
+Once the application is launched the following traces should be visible in the terminal.
+
+.. code-block:: bash
+
+    root@dbf79a437eb3:/turtlesim_move_ws# ros2 run turtlesim turtlesim_square_move
+    [INFO] [1657870899.585667136] [turtlesim_square_move]: New goal [7.544445 5.544445, 0.000000]
+    [INFO] [1657870901.585656311] [turtlesim_square_move]: Reached goal
+    [INFO] [1657870901.585767260] [turtlesim_square_move]: New goal [7.448444 5.544445, 1.570796]
+    [INFO] [1657870905.685637930] [turtlesim_square_move]: Reached goal
+    [INFO] [1657870905.685753714] [turtlesim_square_move]: New goal [7.466837 7.544360, 1.561600]
+    [INFO] [1657870907.885655744] [turtlesim_square_move]: Reached goal
+    [INFO] [1657870907.885742857] [turtlesim_square_move]: New goal [7.466837 7.544360, 3.132396]
+    [INFO] [1657870911.985655175] [turtlesim_square_move]: Reached goal
+    [INFO] [1657870911.985738726] [turtlesim_square_move]: New goal [5.467175 7.581143, 3.123200]
+    [INFO] [1657870914.085652821] [turtlesim_square_move]: Reached goal
+
+
+Running the DDS Router Cloud
```

Comment:
Maybe move Router before the draw_square? So after launching the ROS2 node the turtle moves.
It may happen that the user expect the terminal output, that will not arrive until router is executed

Reply:
I've move the traces to the end of the tutorial